contrib/fuzz/container_fuzzer2.go

```diff
@@ -0,0 +1,77 @@
+// +build gofuzz
+
+/*
+   Copyright The containerd Authors.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package fuzz
+
+import (
+	"bytes"
+	"context"
+
+	fuzz "github.com/AdaLogics/go-fuzz-headers"
+
+	"github.com/containerd/containerd"
+	_ "github.com/containerd/containerd/cmd/containerd"
+	"github.com/containerd/containerd/cmd/containerd/command"
+	"github.com/containerd/containerd/namespaces"
+)
+
+const (
+	defaultRoot    = "/var/lib/containerd"
+	defaultState   = "/tmp/containerd"
+	defaultAddress = "/tmp/containerd/containerd.sock"
+)
+
+func init() {
+	args := []string{"--log-level", "debug"}
+	go func() {
+		// This is similar to invoking the
+		// containerd binary.
+		// See contrib/fuzz/oss_fuzz_build.sh
+		// for more info.
+		command.StartDaemonForFuzzing(args)
+	}()
+}
+
+func fuzzContext() (context.Context, context.CancelFunc) {
+	ctx, cancel := context.WithCancel(context.Background())
+	ctx = namespaces.WithNamespace(ctx, "fuzzing-namespace")
+	return ctx, cancel
+}
+
+func FuzzContainer2(data []byte) int {
```

Comment:
Regarding this function and the filename, I'd like to include `import` (e.g. FuzzContainerImport) rather than `2`.